### DISCUSSION TITLE
Incorporating track-level covariates

### DISCUSSION THREAD

**Discussion by peekaypee** [jmsigner/amt#122] [Q&A]
Hello, could you please help me with a very basic question about SSF? 

I am interested in incorporating covariates that remain the same across tracks; year or group information, for example. Is there a way to do that in the ssf model? I would also like to test if some of these covariates interact with certain others. Currently, the results only show NA. Is there something I am missing?

**Comment by jmsigner** [comment]
Could you give an example? I am not sure, that I fully understand what you are trying to do.
